docs/customization/model.rst

```diff
@@ -94,7 +96,21 @@ The file should be placed in ``AppBundle/Resources/config/doctrine/Country.orm.y
                 classes:
                     model: AppBundle\Entity\Country
 
-4. Additionally if you want to give the administrator an ability to add a ``flag`` to any of Countries,
+4. Update tha database:
+
+* via schema:
```

Comment:
I found `via schema` misleading. It is only a key word of command. Direct database table alter?